### ISSUE TITLE
SPEC: Add a new key in the root of the config file for services

### ISSUE THREAD

**Issue by xmnlab** [osl-incubator/sugar#61]
Add a new key in the root of the config file for services. Example:

```
version: 1.0.0
compose-app: docker-compose
env-file: .env
services:
  default: service1,service3
  list:
    - name: service1
      health-check: true
    - name: service2
      health-check: false
    - name: service3
      health-check: true
```

when the services item is given, `groups` should not be given, otherwise it should raise an error.

**Comment by xmnlab** [comment]
example of an error message for a spec conflict for groups and services:

```
`services` and `groups` flag given. Just use one of them is allowed.
```

**Comment by joel5vega** [comment]
**
![image](https://github.com/osl-incubator/sugar/assets/27688744/d7c2fe12-1947-492b-83cc-2bded2cdebda)
**
Hi @xmnlab , I am trying to use this logic, is it ok?

**Comment by xmnlab** [comment]
not sure ... it is not really clear what it is doing

**Comment by xmnlab** [comment]
fixed by #110 . thanks @joel5vega
